examples/jupyter/bin/marathon_submit.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+curl -k -i -L -X POST -H "Content-type: application/json" "https://$MARATHON/v2/apps" -d@"$@"
```

Comment:
You will also need a `-u username:password` here for clusters with Marathon auth. You might also consider hardcoding this as deploying `jupyter.json`, e.g. `-d@jupyter.json`.